src/types/BaseField.ts

```diff
@@ -18,25 +23,22 @@ export enum BaseFieldScope {
 
 interface BaseField {
 	readonly id: number;
-	label: string;
-	description: string;
 	shortCode: string;
 	dataType: BaseFieldDataType;
 	scope: BaseFieldScope;
+	readonly localizations: BaseFieldLocalization[];
 	readonly createdAt: string;
 }
 
 type WritableBaseField = Writable<BaseField>;
 
+type WritableBaseFieldWithLocalizations = WritableBaseField & {
+	localizations: WritableBaseFieldLocalizationWithBaseFieldContext[];
+};
+
 const writableBaseFieldSchema: JSONSchemaType<WritableBaseField> = {
 	type: 'object',
 	properties: {
-		label: {
-			type: 'string',
-		},
-		description: {
```

Comment:
Oh snap great call I completely forgot about descriptions when we were discussing localization!